pipeline/git/git.go

```diff
@@ -108,7 +153,7 @@ func gitLog(refs ...string) (string, error) {
 func getInfo() (tag, commit string, err error) {
 	tag, err = cleanGit("describe", "--tags", "--abbrev=0")
 	if err != nil {
-		return
+		log.Printf("Failed to retrieve current tag: %s", err.Error())
```

Comment:
Because I've moved the error handling into the Run method for this special case. A repository not having a tag should not prevent retrieving the last commit ID :)